bilby/gw/prior.py

```diff
@@ -1149,6 +1149,14 @@ def from_envelope_file(envelope_file, minimum_frequency,
             frequency median-amplitude median-phase -1-sigma-amplitude
             -1-sigma-phase +1-sigma-amplitude +1-sigma-phase
 
+        There are two definitions of the calibration correction in the
+        literature, one defines the correction as mapping calibrated strain
+        to theoretical waveform templates (:code:`data`) and the other as
+        mapping theoretical waveform templates to calibrated strain
+        (:code:`template`). Prior to version XYZ, :code:`template` was assumed,
```

Comment:
This line is missing the version number. Once that is fixed I am happy to approve.

```suggestion
        (:code:`template`). Prior to version 1.4.0, :code:`template` was assumed,
```